### DISCUSSION TITLE
IoSerder does not work

### DISCUSSION THREAD

**Discussion by jenov** [terjeio/ioSender#173] [General]
Hello good:
The last compilation does not work for me. It tells me that there is a configuration file and it does nothing else. Neither in windows 10, nor in 11.
Sorry I do not speak English

Nombre del dispositivo	DESKTOP-ID4JA6E
Procesador	Intel(R) Core(TM) i5-8300H CPU @ 2.30GHz   2.30 GHz
RAM instalada	8,00 GB (7,85 GB usable)
Identificador de dispositivo	804D0E5E-0E54-42F9-99C8-EF00B09728D6
Id. del producto	00325-96513-02996-AAOEM
Tipo de sistema	Sistema operativo de 64 bits, procesador basado en x64

**Comment by terjeio** [comment]
You have to answer _Yes_ to this question or copy a config file from a previous installation:

![image](https://user-images.githubusercontent.com/20260062/145177905-0b039a82-ebb1-4865-b0f9-3e487cb34a94.png)

Did you do that?

**Comment by jenov** [comment]
I don't know what happened now, it works perfectly in both of us.

no se que paso ahora funciona.Perfectamente en los dos .
